swerve100/src/main/java/org/team100/lib/motor/turning/NeoTurningMotor.java

```diff
@@ -33,10 +45,17 @@ public void set(double output) {
         m_motor.set(output);
     }
 
-    //THIS DOES NOT ACTUALLY SET PID This is just here for the CAN turning motor
-    public void setPID(ControlMode control, double output) {
+    public void setPIDVelocity(double output) {
+        double motorGearing = 1;//I believe there are not any robotics that actually use a neo turning motor I was changing this class to be more ready in case we do
```

Comment:
you shouldn't really write code in advance of needing it; if nothing uses this, then you can leave this stuff out.  just throw UnsupportedOperationException.